packages/acceptance-tests/pkg-tests-specs/sources/commands/up.test.ts

```diff
@@ -128,5 +128,41 @@ describe(`Commands`, () => {
         });
       }),
     );
+
+    test(
+      `it should skip build scripts when using --mode=skip-build`,
+      makeTemporaryEnv({
+        dependencies: {
+          [`no-deps-scripted`]: `1.0.0`,
+        },
+      }, async ({path, run, source}) => {
+        const {stdout} = await run(`up`, `no-deps-scripted`, `--mode=skip-build`, {
+          env: {
+            YARN_ENABLE_INLINE_BUILDS: `1`,
+          },
+        });
+
+        expect(stdout).not.toContain(`must be built`);
+        expect(stdout).toMatchSnapshot();
+      }),
+    );
+
+    test(
+      `it should skip build scripts when using --mode=skip-build (recursive)`,
+      makeTemporaryEnv({
+        dependencies: {
+          [`no-deps-scripted`]: `1.0.0`,
+        },
+      }, async ({path, run, source}) => {
+        const {stdout} = await run(`up`, `no-deps-scripted`, `--recursive`, `--mode=skip-build`, {
+          env: {
+            YARN_ENABLE_INLINE_BUILDS: `1`,
+          },
+        });
+
+        expect(stdout).not.toContain(`must be built`);
+        expect(stdout).toMatchSnapshot();
```

Comment:
Can you update the tests to not use `toMatchSnapshot`? Snapshots make it difficult to tweak the program output, and it's not always clear what's supposed to be important in the snapshot in the first place.

For this kind of thing, better to check for a specific substring (like you did here). Coincidentally I just did it here for `add` and `up`:

https://github.com/yarnpkg/berry/pull/5526/files#diff-ae86f1caf9fde86cd0a9843443ca26a101952fbcf9a385c2cf4bee615738e7e4

Reply:
Done :+1: 

(I originally used snapshots because that's what `install.test.js` used to use)